### PR TITLE
test parallel execution

### PR DESCRIPTION
**Pull request by bschimke95**

### CODE REVIEW

File: .github/workflows/integration.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ jobs:
       - name: Setup Python
         uses: actions/setup-python@v5
         with:
-          python-version: '3.8'
+          python-version: '3.10'
```
Comment (addyess) [inline .github/workflows/integration.yaml:85]: I can get behind running the tests with `3.10` or even `3.12`.  What did you run into with 3.8 that didn't work?

File: .github/workflows/integration.yaml [hunk 2]
```diff
@@ -103,6 +103,7 @@ jobs:
           TEST_LXD_IMAGE: ${{ matrix.os }}
           TEST_INSPECTION_REPORTS_DIR: ${{ github.workspace }}/inspection-reports
         run: |
+          python --version
```
Comment (addyess) [inline .github/workflows/integration.yaml:106]: running pytest through `tox -e integration` should be printing the version right?

File: tests/integration/requirements-test.txt [hunk 3]
```diff
@@ -1,4 +1,5 @@
 coverage[toml]==7.2.5
+pytest-xdist[psutil]==3.6.1
```
Comment (addyess) [inline tests/integration/requirements-test.txt:2]: interesting -- i'm starting to see what's happening here...

**Review by addyess — COMMENTED** [review]
Just watching what's happening here

**Comment by bschimke95** [comment]
This does not work due to eBPF working on kernel level and messing up multiple deployments on the same host.

**Pull request closed without merging**